src/app/photo-explore/explore/explore.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ExploreComponent } from './explore.component';
+
+describe('ExploreComponent', () => {
+  let component: ExploreComponent;
+  let fixture: ComponentFixture<ExploreComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ ExploreComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
```

Comment:
As stated in review above, can we declare beforeEach statement once, just for code clarification?